<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UncertaintyWidget</class>
 <widget class="QWidget" name="UncertaintyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UncertaintyWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QFrame" name="frameLonPlot"/>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="frameDepthPlot"/>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frameLatPlot"/>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frameRmsPlot"/>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="framePhasePlot"/>
      </item>
      <item row="1" column="2">
       <widget class="QFrame" name="frameMagPlot"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frameToolBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>65</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
